### ISSUE TITLE
Error when using Entries in Repeater | Tailor

### ISSUE THREAD

**Issue by ogalbnafets** [octobercms/october#5684]
Hello!
 I am experimenting with tailor and its working great so far.

However, if I want to include an entry-selector in a repeater, I get the Error

> Call to undefined method Tailor\Models\RepeaterItem::getBlueprintDefinition()" on line 71 of /var/www/stwh/ocms-stage/vendor/laravel/framework/src/Illuminate/Support/Traits/ForwardsCalls.php

Mixin:
```
handle: Wochenkarte\Speisendefinition
type: mixin
name: Speisendefinition

fields:
    speisen:
        tab: Speisen
        type: repeater
        form:
            fields:                         
                Allergene:
                    label: Allergene
                    type: entries
                    source: Speisen\Allergien      
```

Source:
```
handle: Speisen\Allergien
type: stream
name: Allergien
drafts: true

navigation:
    icon: icon-bookmark
    parent: Wochenkarte\Tagesmenu
    order: 10

fields:
    Beschreibung:
        tab: Bearbeiten
        label: Allergienname
        type: textarea
        size: small

    Kurz:
        tab: Bearbeiten
        label: Kurzbezeichnung
        type: textarea
        size: small

    Icon:
        tab: Icon
        label: Icon
        type: mediafinder
        mode: image
        span: adaptive    

```

Did I miss something?
Thank you!

**Comment by daftspunk** [comment]
> Did I miss something?

I think you've found a bug, if you set `maxItems: 1` then it should work. However, using multiple items appears to be unsupported. We will fix it in v3.2 and let you know here once the feature is implemented.

**Comment by ogalbnafets** [comment]
Thank you very much!

**Comment by daftspunk** [comment]
Confirmed this is fixed in v3.2 and this version will be available in approximately 1 week.
